documentation/src/main/wiki/FallbackHandling.md

```diff
@@ -0,0 +1,134 @@
+# Fallback Handling
+
+Sometimes during Fragment processing things can go wrong. There can be multiple reasons: misconfiguration, programming 
+error, lack of service availability. Knot.x gives you the ability to define a fallback strategy that is applied when 
+dynamic fragment processing fails.
+
+## How does it work? 
+
+When no fallback is defined an error that occurs during processing of a single fragment will trigger a 500 error 
+response from the server. This will basically stop the page rendering process. 
+
+You have the option to alter this behavior. Knots have the option to mark Fragments with enabled fallback as 
+failed - and continue the rendering process. 
+
+Failed fragments are ignored by [[Handlebars Knot|HandlebarsKnot]]. 
+
+Failed fragments are not rendered by the [[Assembler|Assembler]]. Instead of rendering content of the failed fragment 
+[[Assembler|Assembler]] will apply defined fallback strategy. 
+- you may use `BLANK` fallback strategy - in such case failed snippets will not be rendered at all.
+- you may provide your own markup that will replace the failed snippet
+- you may configure this behavior for each snippet or globally           
+
+## How to configure? 
+
+You may configure fallback behavior for each snippet - or you may use global configuration to apply fallback to all 
+snippets in your application. Local configuration (on snippet level) takes precedence over the global one. 
+    
+### Simple blank fallback (per snippet)
+To render an empty string instead of your failed snippet add `data-knotx-fallback="BLANK"` attribute:
+```
+<knotx-snippet data-knotx-knots="databridge,handlebars" 
+  data-knotx-databridge-name="unstable-service"
+  data-knotx-fallback="BLANK">
+      {{#if _result}}<h2>{{_result.count}}</h2>{{/if}}
+</knotx-snippet>
+```
+
+### Static markup fallback (per snippet)
+To render a static markup instead of your failed snippet: 
+1. create a fallback snippet with `data-knotx-fallback-id` attribute 
+2. point to it in your snippet
+```
+<knotx-snippet data-knotx-knots="databridge,handlebars" 
+  data-knotx-databridge-name="unstable-service"
+  data-knotx-fallback="CUSTOM">
+      {{#if _result}}<h2>{{_result.count}}</h2>{{/if}}
+</knotx-snippet>
+<knotx:fallback data-knotx-fallback-id="CUSTOM">
```

Comment:
What do you think about changing `CUSTOM` to e.g. `MY_FALLBACK_ID`.
That might be more clear, that `CUSTOM` is not the mandatory id part for the custom fallback snippet.

Reply:
Changed

Reply:
also: knotx-stack and knotx-example-project have PRs open that showcase this feature.